backend/generator.js

```diff
@@ -18,8 +18,12 @@ exports.executeScript = function (socket, formData) {
   var uniqueId = uuidV4();
   var webUI = "true";
   var subProject = "";
+  var subDocpath = [];
   if (formData.subProject !== undefined) {
-    subProject = formData.subProject.join(",")
+    subProject = formData.subProject.join(",");
+    for(i=0; i<formData.subProject.length; i++){
+        subDocpath.push('docs');
+    }
```

Comment:
are there any ways to handle this is the script? because right now we handling all the defaults in the script only.